internal/handle.go

```diff
@@ -600,3 +606,29 @@ func (h *Handle) UpdateLatest(number uint32) error {
 	}
 	return nil
 }
+
+// Returns a copy of the trusted umbilical roots.
+//
+// Requires write lock on mux.
```

Comment:
Why not a read lock?

Reply:
Because two readers might try to set `h.UmbilicalRoots` at the same time.